os/kernel/sched/sched.h

```diff
@@ -272,9 +264,12 @@ void sched_timer_reassess(void);
 #define sched_timer_reassess()
 #endif
 
-#if defined(CONFIG_SCHED_CPULOAD) && !defined(CONFIG_SCHED_CPULOAD_EXTCLK)
+#ifdef CONFIG_SCHED_CPULOAD
+#ifndef CONFIG_SCHED_CPULOAD_EXTCLK
 void weak_function sched_process_cpuload(void);
 #endif
+void sched_clear_cpuload_pid(pid_t pid);
```

Comment:
1. When `CONFIG_SCHED_CPULOAD_EXTCLK` is enabled, is this necessary?
2. Why is `sched_process_cpuload()` in `CONFIG_SCHED_CPULOAD_EXTCLK` conditional even its function is not in that conditional?